homeassistant/components/zimi/quality_scale.yml

```diff
@@ -0,0 +1,26 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: >
+      There are no service actions.
+  appropriate-polling:
+    status: done
+    comment: >
+      There is no polling of the entities.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
I think I checked this already, but where was the code hosted? The code needs to be in a public repo, with an OSI approved license, named releases and automatic releases (CICD(

Reply:
See https://mark_hannon@bitbucket.org/mark_hannon/zcc.git and pypi.

I will add a comment in the file.